src/Lighthouse/App.config

```diff
@@ -20,13 +20,13 @@
 						
 						remote {
 							log-remote-lifecycle-events = DEBUG
-							helios.tcp {
-								transport-class = "Akka.Remote.Transport.Helios.HeliosTcpTransport, Akka.Remote"
+							dot-netty.tcp {
+								transport-class = "Akka.Remote.Transport.DotNetty.TcpTransport, Akka.Remote"
 								applied-adapters = []
 								transport-protocol = tcp
 								#will be populated with a dynamic host-name at runtime if left uncommented
-								#public-hostname = "POPULATE STATIC IP HERE"
-								hostname = "0.0.0.0"
+								public-hostname = "172.22.144.2"
```

Comment:
When using Docker to host Lighthouse, you have to call `docker run -di --ip=**172.22.144.2** --rm --name lighthouse nickc/lighthouse:latest` with the same static ip that is set here.  I've left this IP address since it works on the default Windows Container NAT.